src/components/PlayerChip.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+interface PlayerChipProps {
+  first_name: string;
+  last_name: string;
+  number: number;
+}
+
+const PlayerChip = ({ first_name, last_name, number }: PlayerChipProps) => {
+  const color = selectColor(first_name);
+  return (
+    <div
+      className={`flex items-center text-black p-8 space-x-12 w-80 h-16 rounded-lg border border-black`}
+    >
+      <div
+        style={{ backgroundColor: color }}
+        className="rounded-full py-2 px-4"
+      >
+        #{number}
+      </div>
+      <div style={{ color: color }} className="font-medium w-56">
```

Comment:
Instead of passing in `color: color` here you can just type `{{ color }}` because the variable name matches the attribute.